tests/per_usage/divs_html_elements/scripts/awesome-doc-styling.js

```diff
@@ -0,0 +1,26 @@
+let is_stylished = false;
+function toggle_style() {
+  const stylished_classname = "stylished";
+
+  let apply_toggle_style = is_stylished
+    ? function (elements) {
+        elements.removeClass(`${stylished_classname}`);
+      }
+    : function (elements) {
+        elements.addClass(`${stylished_classname}`);
+      };
+  is_stylished = !is_stylished;
+
+  let elements = $("body").find("code, div.fragment, pre.fragment, main.contents");
+  apply_toggle_style(elements);
+}
+
+let is_small = false;
+function toggle_small() {
```

Comment:
`toggle_small` -> `toggle_small_style`

Right now, this function only works if `toggle_style` was called before.

An improvement here would be to add a simple if-then here, such as :

```js
if (!is_stylished)
   toggle_style()
```

and modify `toggle_style` so when L7 it removes the classes, then it also reset sizes

L7 :
```js
function (elements) {
        elements.removeClass(`${stylished_classname}`);
        if (is_small)
            toggle_small()
      }

```